jcache/commons/src/test/java/org/infinispan/jcache/util/InMemoryJCacheLoader.java

```diff
@@ -17,10 +16,22 @@
  */
 public class InMemoryJCacheLoader<K, V> implements CacheLoader<K, V> {
 
-   private final ConcurrentMap<K, V> store = CollectionFactory.makeConcurrentMap();
+   private final ConcurrentMap<K, V> store;
 
    private final LongAdder counter = new LongAdder();
 
+   public InMemoryJCacheLoader(ConcurrentMap<K, V> store) {
+      this.store = store;
+   }
+
+   public static <K, V> InMemoryJCacheLoader<K, V> create() {
+      return create(new ConcurrentHashMap<>());
+   }
+
+   public static <K, V> InMemoryJCacheLoader<K, V> create(ConcurrentMap<K, V> store) {
```

Comment:
Factory methods help with avoiding the need for multiple constructors which is a pain to deal with, particularly if subclassing.